### PR TITLE
Desktop

### PR DESCRIPTION
**Pull request by Blossom-Aniedi**
- I made sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- I made sure that I used the correct ([Github flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- I also made sure that I documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

### CODE REVIEW

**Review by mmsesay — COMMENTED** [review]
Hi @Blossom-Aniedi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't meet the following requirements:
 - [ ] No descriptive PR summary. It will be good to always provide a descriptive PR summary, explaining the changes that were introduced in your project. That will give other developers or visitors a brief of what you worked on.
 - [ ] The README should be descriptive also of your project. Please use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) as a guide to work on your readme. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -366,3 +383,342 @@ footer {
...
+
+  .connect {
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    margin-top: 24px;
+  }
+
+  .card5 {
+    width: 1100px;
```
Comment (wuletawwonte) [inline style.css]: - [ ] The width of the website is wider than the width of the viewport when the viewport size is in the range 768px - 1024px. The website starts to scroll horizontally when its content is wider than the viewport. Please make sure everything fits within the size of the viewport. You can use `%` or `vw` for horizontal margins and padding as well. And please avoid giving static values in `px` for a width of a container.  

Screen Shot 2022-07-08 at 21 21 59

File: README.md [hunk 2]
```diff
@@ -1,97 +1,50 @@
-# HTML & CSS3 Course
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-If you are not familiar with linters, read [root level README](../README.md).
+# Portfolio Website
```
Comment (wuletawwonte) [inline README.md:3]: - [ ] Please add a description of the changes you made in this pull request in the details section of your pull request. When a description is added it's going to appear in the space pointed by the arrow in the following picture. 

imageedit_4_3738492840

**Review by wuletawwonte — CHANGES_REQUESTED** [review]
# Hi @Blossom-Aniedi 👋🏿👋🏿👋🏿  

Good job so far!
There are some issues that you still need to work on to go to the next project! You have only a few more steps and you are done! 💪 💪 💪 

giphy

## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by prolajumokeoni — APPROVED** [review]
Hi @Blossom-Aniedi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**